libs/shared-web/src/constants/unprotected-routes.const.ts

```diff
@@ -0,0 +1,12 @@
+import { ROUTES } from './routes.enum';
+
+export const UNPROTECTED_ROUTES: ROUTES[] = [
+  ROUTES.AUTH_SIGNUP,
+  ROUTES.AUTH_LOGIN,
+  ROUTES.AUTH_RESET_REQUEST,
+  ROUTES.AUTH_RESET_TOKEN,
+  ROUTES.AUTH_INVITATION_TOKEN,
+  ROUTES.AUTH_APPLICATION,
+];
```

Comment:
❓ question: Are there any other routes that belong here?‏

Reply:
Is there a chance if somehow doing this without the need to define unprotected routes here? What is the current problem we are solving with this list? (wondering if there is another way around it)

Reply:
@scopsy Refactored! This was due to the initial constraints with the ordering of the providers -- LD was wrapped around the auth provider, so we didn't necessarily have the context available.

Since I switched the order of the tree in this PR, I updated it to use the logged-in status!